### PR TITLE
Add a warning in exporting-grafana-snapshots.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
@breeswish @qqqdan

**Review by breeswish — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

File: exporting-grafana-snapshots.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ title: 将 Grafana 监控数据导出成快照
 summary: 了解如何将 Grafana 监控数据导出为快照以及如何将快照文件可视化。
 ---
 
+> **警告：**
+>
+> 从 TiDB v6.0.0 起，PingCAP 不再维护 MetricsTool 工具。从 v6.1.0 起，不再提供 MetricsTool 的文档。
```
Comment (TomShawn) [inline exporting-grafana-snapshots.md:8]: 这里有「v6.1.0」的表述，是否视为 v6.1.0 的文档呢？如果要及时向用户传达「从 TiDB v6.0.0 起，PingCAP 不再维护 MetricsTool 工具。」这个信息，是否可以在这个 PR 里先去掉「v6.1.0 的信息」呢？等 6.1 发版前再加上。

File: exporting-grafana-snapshots.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ title: 将 Grafana 监控数据导出成快照
...
+> 如需导出监控数据，建议使用 [PingCAP Clinic 诊断服务](/clinic/clinic-introduction.md)，支持一键导出诊断 TiDB 集群时所需要的信息，包括监控数据、日志、集群拓扑、配置、参数等。
```
Comment (en-jin19) [inline exporting-grafana-snapshots.md:9]:
```suggestion
> 如需导出监控数据，建议使用 [PingCAP Clinic 诊断服务](/clinic/clinic-introduction.md)一键导出诊断 TiDB 集群时所需要的信息，包括监控数据、日志、集群拓扑、配置、参数等。
```

File: exporting-grafana-snapshots.md (hunk 1 above)
Comment (en-jin19) [inline exporting-grafana-snapshots.md:8]:
```suggestion
> 从 TiDB v6.0.0 起，PingCAP 不再维护 MetricsTool 工具。
```

Comment (en-jin19) [reply exporting-grafana-snapshots.md:8]: 已在下方 comments 中进行修改， @lilyjazz @TomShawn PTAL，谢谢。

**Pull request merged**